src/main/java/mate/academy/spring/controller/UserController.java

```diff
@@ -22,7 +23,7 @@ public UserController(UserService userService, UserMapper userMapper) {
     }
 
     @GetMapping("/by-email")
-    public UserResponseDto getByEmail(@RequestParam String email) {
+    public UserResponseDto getByEmail(@Valid @RequestParam String email) {
```

Comment:
What `@Valid` should do here?
Should we use `@Email` for input email?

Reply:
changed to @email. In @Target{Parametr}